coalib/output/ConsoleInteraction.py

```diff
@@ -864,20 +864,19 @@ def show_enumeration(console_printer,
 def show_bear(bear,
               show_description,
               show_params,
-              console_printer):
+              console_printer,
+              args=None):
     """
     Displays all information about a bear.
 
     :param bear:             The bear to be displayed.
     :param show_description: True if the main description should be shown.
     :param show_params:      True if the details should be shown.
     :param console_printer:  Object to print messages on the console.
+    :param args:             Args passed to coala command.
```

Comment:
It would be better if you used "arguments" instead of "args" in the description :)

Reply:
But the parameter is called `args` :O